README.md

```diff
@@ -7,6 +7,33 @@
 
 > The TypeScript Definition Manager.
 
+## Notice Regarding TypeScript@2.0
+
+**For users doing `typings install dt~<package> --global` and receiving error**
+
+Starting from `TypeScript@2.0`, TypeScript user can acquire typings through `npm install @types/<package>`.
+
+Those typings are coming from the [`type-2.0`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/types-2.0) branch of [DefinitlyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped).
```

Comment:
They used to be from `types-2.0` to avoid breaking `master`.
I recall when I make a change, they asked me to check it in the `types-2.0` branch instead.